lib/resque/scheduler/env.rb

```diff
@@ -56,32 +56,22 @@ def setup_pid_file
 
       def setup_scheduler_configuration
         Resque::Scheduler.configure do |c|
-          if options.key?(:app_name)
```

Comment:
I understand, but this code set nil for all the keys.
https://github.com/resque/resque-scheduler/blob/f0bb556167bd297ef3464e80d6b5b9bde0ab3fdd/lib/resque/scheduler/cli.rb#L139